common/djangoapps/student/tests/test_models.py

```diff
@@ -244,6 +256,169 @@ def test_enrollments_not_deleted(self):
         self.assertEqual(enrollment_refetched.all()[0], enrollment)
 
 
+@override_waffle_flag(REDIRECT_TO_COURSEWARE_MICROFRONTEND, active=True)
+@override_waffle_flag(COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES, active=True)
+@override_waffle_flag(COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES_STREAK_CELEBRATION, active=True)
+class UserCelebrationTests(SharedModuleStoreTestCase):
+    """
+    Tests for User Celebrations like the streak celebration
+    """
+    @classmethod
+    def setUpClass(cls):
+        super(UserCelebrationTests, cls).setUpClass()
+        cls.course = CourseFactory()
+        cls.course_key = cls.course.id  # pylint: disable=no-member
+
+    def setUp(self):
+        super(UserCelebrationTests, self).setUp()  # lint-amnesty, pylint: disable=super-with-arguments
+        self.user = UserFactory()
+        self.request = mock.Mock()
+        self.request.user = self.user
+        CourseEnrollmentFactory(course_id=self.course.id)  # pylint: disable=no-member
+
+    def test_first_check_streak_celebration(self):
+        should_celebrate = UserCelebration.perform_streak_updates(self.user, self.course_key)
+        today = datetime.datetime.now(UTC).date()
+        self.assertEqual(self.user.celebration.first_day_of_streak, today)
```

Comment:
Minor nit, I don't super care. But apparently we are trying to move towards bare asserts, rather than the `assertX()` methods. So like, this would become `assert self.user.celebration.first_day_of_streak == today`. Might be good to use that style in new code at least.